src/content/structured/components/side-nav/code.mdx

```diff
@@ -747,6 +749,115 @@ export const withReactRouter = [
   </MemoryRouter>
 </ComponentPreview>
 
+### With MUI icons in SlottedSVG
+
+export const withMUIicons = [
+  {
+    language: "React",
+    snippet: `<MemoryRouter initialEntries={["/"]}>
+  <div style={{ display: "flex", flexDirection: "row"}}>
+    <IcSideNavigation
+      appTitle="ACME"
+      version="v0.0.0"
+      status="BETA"
+    >
+      <SlottedSVG slot="app-icon" ... />
+      <IcNavigationItem
+        slot="primary-navigation"
+      >
+        <NavLink to="/" slot="navigation-item">
+          <SlottedSVG slot="icon" ... />
+          Home
+        </NavLink>
+      </IcNavigationItem>
+      <IcNavigationGroup label="Navigation Grouped" slot="primary-navigation" expandable="true">
+        <IcNavigationItem>
+          <NavLink to="/child-item" slot="navigation-item">
+            <SlottedSVG slot="icon" ... />
+            Home
+          </NavLink>
+        </IcNavigationItem>
+        <IcNavigationItem>
+          <NavLink to="/child-item-2" slot="navigation-item">
+            <SlottedSVG slot="icon" ... />
+            Daily Tippers
+          </NavLink>
+        </IcNavigationItem>
+      </IcNavigationGroup>
+      <IcNavigationItem
+        slot="secondary-navigation"
+      >
+        <NavLink to="/daily-tippers" slot="navigation-item">
+          <SlottedSVG slot="icon" ... />
+            Daily Tippers
+        </NavLink>
+      </IcNavigationItem>
+    </IcSideNavigation>
+    <main>
+      <IcSectionContainer>
+        <Routes>
+          <Route path="/" element={() => <IcTypography>This is the home page</IcTypography>} />
+          <Route path="/daily-tippers" element={() => <IcTypography>This page is about daily tippers</IcTypography>} />
+        </Routes>
+      </IcSectionContainer>
+    </main>
+  </div>
+</MemoryRouter>
+    `,
+  },
+];
+
+<ComponentPreview
+  snippets={snippetsDefault}
```

Comment:
We have the code example above in a variable called `withMUIicons` but we haven't passed it through to the `ComponentPreview`

Reply:
The variable should be `withMDIicons` instead of `withMUIicons`